_posts/2021-04-23-timestamps.markdown

```diff
@@ -23,6 +23,7 @@ Timestamps are required for **all selling posts on the _subreddit_.** Timestamps
     - The only exception is sealed items - these must clearly show the seal on the box as well as the timestamp.
 - Show the exact model number of the item you're selling if possible.
 - Only one of the images in a collection of images needs to be timestamped, as long as it meets the requirements.
+- [Example Timestamp](https://i.imgur.com/oOkHF3d.jpg)
```

Comment:
Please create an `img` directory inside `assets`, and use an image embed like so (note the exclamation point at the start of the line):
```
![Alt text goes here](Image URL goes here)
```

You can use a relative link to items in the assets folder by setting the URL to `/assets/img/whatever.png`.

(also don't use JPEGs for images which contain primarily text)